opencraft/settings.py

```diff
@@ -596,3 +600,22 @@
 AWS_ACCESS_KEY_ID = env('AWS_ACCESS_KEY_ID', default=None)
 AWS_S3_BUCKET_PREFIX = env('S3_BUCKET_PREFIX', default='ocim')
 AWS_IAM_USER_PREFIX = env('IAM_USER_PREFIX', default='ocim')
+
+# Consul ######################################################################
+
+# The encryption key used to gossip in a Consul cluster.
+CONSUL_ENCRYPT = env('CONSUL_ENCRYPT', default='')
+
+# The list of server agents in the Consul cluster.
+CONSUL_SERVERS = env.list('CONSUL_SERVERS', default=[])
+
+# Filebeat ####################################################################
+
+# The Logstash host to forward logs to.
+FILEBEAT_LOGSTASH_HOSTS = env.list('FILEBEAT_LOGSTASH_HOSTS', default=[])
+
+# The URL to retrieve the public CA certificate from.
+FILEBEAT_CA_CERT_URL = env('FILEBEAT_CA_CERT_URL', default='')
+
+# Common fields for all Filebeat prospectors.
+FILEBEAT_COMMON_PROSPECTOR_FIELDS = env.dict('FILEBEAT_COMMON_PROSPECTOR_FIELDS', default={})
```

Comment:
`env.dict` is weird.  The format is a [comma-separated list of `key=value` pairs](http://django-environ.readthedocs.io/en/latest/#supported-types), without support for quoting special characters.  I think we should use `env.json()` here, and make sure the Ansible code wriites out proper JSON.